e2e/nx-init/src/nx-init-react.test.ts

```diff
@@ -163,7 +163,6 @@ describe('nx init (for React - legacy)', () => {
     );
 
     const packageJson = readJson('package.json');
-    expect(packageJson.devDependencies['@nx/jest']).toBeUndefined();
```

Comment:
jest is the default so if the unitTestRunner is not set "none" this will be used.